packages/spark-core/base/_links.scss

```diff
@@ -4,63 +4,50 @@
 //
 .sprk-b-Link {
   color: $link-color;
-  font-weight: $link-font-weight;
+  font-weight: $link-inline-font-weight;
   position: relative;
   text-decoration: none;
   transition: $link-transition;
+  box-shadow: inset 0 $link-underline-width $link-underline-color;
 
   &:visited {
     color: $link-visited-color;
```

Comment:
inline and standalone links are using this style for visited links (black text)

Reply:
I tried some things this morning in an attempt to help since on Friday you mentioned that we can't change box-shadow on visited state. I tried using CSS vars here https://codepen.io/afebbraro/pen/GwMLxb?editors=1111 but had no luck. The var does change the box-shadow color on :hover but again, not on :visited. :(